QueryAnalyzer/QueryAnalyzer.Domain/IndexProposition.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace QueryAnalyzer.Domain;
+
+public class IndexProposition
+{
+    public string? TableSchema { get; set; } 
```

Comment:
drobna rzecz - string jest nullable z urzędu. chyba że informujesz że się może pojawić tutaj null, to wtedy elegancko